flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/factories/TestValuesTableFactory.java

```diff
@@ -756,6 +763,8 @@ public DynamicTableSink createDynamicTableSink(Context context) {
         final Map<String, DataType> writableMetadata =
                 convertToMetadataMap(
                         helper.getOptions().get(WRITABLE_METADATA), context.getClassLoader());
+        final int[][] targetColumns =
```

Comment:
Why do we get the target columns by table options instead of using `SupportsTargetColumnWriting`?